tests/components/python_script/test_init.py

```diff
@@ -64,7 +64,7 @@ def test_execute_with_data(hass, caplog):
 hass.states.set('test.entity', data.get('name', 'not set'))
     """
 
-    hass.async_add_job(execute, hass, "test.py", source, {"name": "paulus"})
+    hass.async_add_job(execute, hass, "integration_test.py", source, {"name": "paulus"})
```

Comment:
Please don't change these files.

Reply:
this was an unwanted commit. reverted back.